src/ptvsd/_vendored/pydevd/_pydevd_bundle/pydevd_comm.py

```diff
@@ -1076,6 +1076,99 @@ def internal_get_description(dbg, seq, thread_id, frame_id, expression):
         dbg.writer.add_command(cmd)
 
 
+def _get_line_for_traceback(file_path, line_no):
```

Comment:
`linecache.getline` is what traceback itself uses for this exact purpose; I think we should do so as well.